PullUpController/PullUpController.swift

```diff
@@ -59,6 +59,27 @@ open class PullUpController: UIViewController {
         return []
     }
     
+    /**
+     Determines if this pull up controller can go fullscreen
+     */
+    open var pullUpControllerCanGoFullscreen: Bool {
+        return true;
+    }
+    
+    /**
+     Determines if this pull up controller can fully hide below the screen
+     */
+    open var pullUpControllerCanHide: Bool {
+        return true;
+    }
+    
+    /**
+     The Y-velocity for pan events
+     */
+    open var pullUpControllerVelocityY: CGFloat {
```

Comment:
It's not really clear what this property rappresents, could you please rename it in `pullUpControllerSkipPointThresholdVelocityY`? (or something similar)